### PR TITLE
Append /ufe0f to thumbsup and thumsdown reactions

### PR DESCRIPTION
**Pull request by jaller94**
Fixes: #488 

Related: https://github.com/vector-im/element-web/issues/13926

### CODE REVIEW

File: src/BridgedRoom.ts [hunk 1]
```diff
@@ -660,7 +660,11 @@ export class BridgedRoom {
             return;
         }
 
-        const reactionKey = emoji.emojify(`:${message.reaction}:`, getFallbackForMissingEmoji);
+        let reactionKey = emoji.emojify(`:${message.reaction}:`, getFallbackForMissingEmoji);
+        // Element uses the default thumbsup and thumbsdown reactions with an appended variant character.
+        if (reactionKey === '👍' || message.reaction === '👎') {
```
Comment (Half-Shot) [inline src/BridgedRoom.ts:665]:
```suggestion
        if (reactionKey === '👍' || reactionKey === '👎') {
```

Comment (Half-Shot) [reply src/BridgedRoom.ts:665]: Spotted

**Review by Half-Shot — APPROVED** [review]
Otherwise excellent!

File: src/BridgedRoom.ts [hunk 2]
```diff
@@ -660,7 +660,11 @@ export class BridgedRoom {
...
+            reactionKey += '\ufe0f'.normalize(); // VARIATION SELECTOR-16
```
Comment (Cadair) [inline src/BridgedRoom.ts:666]: it's only the :+1: and :-1: emoji Element does this to?!

Comment (jaller94) [reply src/BridgedRoom.ts:666]: I tested the other quick reactions and they work fine. (2 means that Matrix and Slack are sending the same character)
Screenshot_2020-09-24_21-56-39

Comment (Cadair) [reply src/BridgedRoom.ts:666]: From the MSC: 

> When sending emoji reactions, the key field should include the colourful variation-16 when applicable.

why is it only applicable to append VS16 to :+1:  and :-1: ?!

Comment (tulir) [reply src/BridgedRoom.ts:666]: It's not applicable to everything, but I'm 97% sure there's more than :+1:/:-1:

**Pull request merged**